### ISSUE TITLE
Support more commands for multi-select (Inspect, View Logs, etc.)

### ISSUE THREAD

**Issue by bwateratmsft** [microsoft/vscode-docker#1631]
After #1623, there are still some commands for which multi-select is not enabled (e.g. Inspect, View Logs, etc.). We should consider enabling them, but I wanted to create this as a separate issue from #331.

@eine for visibility :smile:

**Comment by bwateratmsft** [comment]
Absent any explicit demand I'll close this for now.
